notes.txt

```diff
@@ -0,0 +1,35 @@
+This branch adds pm-cpu to maint-1.0, but there are still some changes
+required to get tests running.
+
+
+There are build errors with some cases using Intel that can be resolved using a more
+recent xml_stream_parser.c.  The one here in this directory is from a recent version
+of E3SM main.
+
+cp xml_stream_parser.c components/mpas-source/src/framework/xml_stream_parser.c
+
+The build error without this change might be:
+xml_stream_parser.c:1732:6: error: call to undeclared function 'stream_mgr_add_field_c'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
+                                        stream_mgr_add_field_c(manager, streamID, (const char *)fieldname, packages_local, &err);
+                                        ^
```

Comment:
@jonbob can you make the above changes to the MPAS submodule version pointed to by maint-1.0?  Just need intel working on pm-cpu.